misc/config_definitions.json

```diff
@@ -1538,16 +1538,16 @@
             "type": "boolean"
         },
         "fping": {
-            "default": "fping",
+            "default": "/usr/bin/fping",
```

Comment:
Leave the defaults to the unqualified paths.  Binaries are expanded to full paths on first run:
https://github.com/librenms/librenms/blob/master/LibreNMS/Config.php#L442